### ISSUE TITLE
tabline.apply_to_config() crashes if called before tabline.setup()

### ISSUE THREAD

**Issue by JimtotheB** [michaelbrusegard/tabline.wez#37]
If tabline.apply_to_config() is called before tabline.setup(), it will crash due to attempting to access nil table values, as they haven't been set yet. Specifically this call https://github.com/michaelbrusegard/tabline.wez/blob/9e6656b06e438f35d08d7a447318da03aba7a449/plugin/init.lua#L61

**Comment by michaelbrusegard** [comment]
Your tabline shows the Catpuccin Mocha colors, because that is the colors you have defined in your `color_overrides` table. If you want the colors from the theme you have set, in this case GruveboxDark, you should remove the `color_ovverrides`. The `color_overrides` table is meant to overwrite single color changes or provide a custom theme.

**Comment by JimtotheB** [comment]
No no, sorry. You were getting some stream of consciousness posting with my OP. Me Putting in the color overrides was just my attempt to get tabline.apply_to_config() to stop crashing, sorry. The issue with the theme, was it wasn't in options originally, error on my part.. the apply_to_config() is still crashing, and that's what originally brought me here.

<img width="972" alt="image" src="https://github.com/user-attachments/assets/02d0f71a-d0b8-4f6f-a586-3596ba8292c7">
 
I got that error several hours ago when I first set the plugin up, but I an coming to Wezterm from Kitty and had to get my head around how to debug things.  This is the rest of the config above .setup(), I attempted to manually set all of the defaults apply_to_config sets, and it crashes out on the require('tabline.config').... bit.

<img width="928" alt="image" src="https://github.com/user-attachments/assets/84833613-7181-4355-83bc-bcee65276f4c">
